regression/test.pl

```diff
@@ -74,6 +74,7 @@ ($$$$$$$$$$)
   foreach my $key (@keys) {
     my $value = $defines->{$key};
     $options =~ s/(\$$key$|\$$key )/$value /g;
+    $options =~ s/\$$key:/$value:/g;
```

Comment:
For maintainability, I would love a comment to explain why we need the `:` as a separator, e.g.
```
# Variables in --classpath are separated by colons
```